specification/batch/data-plane/Microsoft.Batch/stable/2021-01-01.13.0/BatchService.json

```diff
@@ -13636,6 +13930,187 @@
+          },
+          {
+            "value": "zonal",
+            "description": "Nodes in the pool will be spread across different availability zones with best effort balancing.",
+            "name": "zonal"
+          }
+        ]
+      }
+    },
+    "NodePlacementConfiguration": {
+      "properties": {
+        "policy": {
+          "$ref": "#/definitions/NodePlacementPolicyType",
+          "title": "Node placement Policy type on Batch Pools.",
+          "description": "Allocation policy used by Batch Service to provision the nodes. If not specified, Batch will use the regional policy."
+        }
+      }
+    },
+    "VMExtension": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "title": "The name of the virtual machine extension."
+        },
+        "publisher": {
+          "type": "string",
+          "title": "The name of the extension handler publisher."
+        },
+        "type": {
+          "type": "string",
+          "title": "The type of the extensions."
+        },
+        "typeHandlerVersion": {
+          "type": "string",
+          "title": "The version of script handler."
+        },
+        "autoUpgradeMinorVersion": {
+          "type": "boolean",
+          "description": "Indicates whether the extension should use a newer minor version if one is available at deployment time. Once deployed, however, the extension will not upgrade minor versions unless redeployed, even with this property set to true."
+        },
+        "settings": {
+          "type": "object",
+          "title": "JSON formatted public settings for the extension."
+        },
+        "protectedSettings": {
+          "type": "object",
+          "description": "The extension can contain either protectedSettings or protectedSettingsFromKeyVault or no protected settings at all. "
+        },
+        "provisionAfterExtensions": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "title": "The collection of extension names.",
+          "description": "Collection of extension names after which this extension needs to be provisioned."
+        }
+      },
+      "required": [
+        "name",
+        "publisher",
+        "type"
+      ],
+      "title": "The configuration for virtual machine extensions."
+    },
+    "InstanceViewStatus": {
+      "properties": {
+        "code": {
+          "type": "string",
+          "title": "The status code."
+        },
+        "displayStatus": {
+          "type": "string",
+          "title": "The localized label for the status."
+        },
+        "level": {
+          "type": "string",
+          "title": "Level code.",
+          "enum": [
+            "error",
+            "info",
+            "warning"
+          ],
+          "x-ms-enum": {
+            "name": "StatusLevelTypes",
+            "modelAsString": false,
+            "values": [
+              {
+                "value": "error"
```

Comment:
Need description for each enum.

Reply:
This is a pass-through to VMSS and matches the swagger there (also the swagger for non-VMSS VM extensions)